src/build/content/static.ts

```diff
@@ -27,21 +27,23 @@ export const copyStaticContent = async (ctx: PluginContext): Promise<void> => {
     })
 
     const fallbacks = ctx.getFallbacks(await ctx.getPrerenderManifest())
+    const fullyStaticPages = await ctx.getFullyStaticHtmlPages()
 
     try {
       await mkdir(destDir, { recursive: true })
       await Promise.all(
         paths
-          .filter((path) => !paths.includes(`${path.slice(0, -5)}.json`))
+          .filter((path) => !path.endsWith('.json') && !paths.includes(`${path.slice(0, -5)}.json`))
```

Comment:
just small optim to return false early for json and avoid array search, doesn't change results, because if `path` ends with `.json` then paths will also include `${path.slice(0, -5)}.json` as it will be same path